### PR TITLE
Setting up a README

### PR DESCRIPTION
**Pull request by chtakahashi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# security-hub-collector-ecs-runner
+
+This repo contains a Terraform module which will deploy a scheduled ECS
+task which can run a periodic InSpec scan against an AWS account. The module supports following features:
```
Comment (LinuxBozo) [inline README.md]: This runs inspec?

**Review by LinuxBozo — CHANGES_REQUESTED** [review]
Should also mention or link to security-hub-collector somewhere in here

File: README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  ecs_subnet_ids = [data.aws_subnet.private_a.id]
+
+  schedule_task_expression  = "cron(30 9 * * ? *)"
+  logs_cloudwatch_group_arn = aws_cloudwatch_log_group.main.arn
+  ecs_cluster_arn           = "arn:aws:ecs:us-east-1:037370603820:cluster/aws-scanner-inspec"
+
+  output_path       = ""
+  s3_results_bucket = aws_s3_bucket.security_hub_collector.bucket
+  s3_key            = ""
+  team_map          = filebase64("${path.module}/teammap.json")
```
Comment (LinuxBozo) [inline README.md]: Might want to mention that is is required to be base64'd

File: README.md (hunk 1 above)
Comment (chtakahashi) [reply README.md]: wait, why didn't this commit?

**Review by LinuxBozo — APPROVED** [review]

**Pull request merged**